docs/en/UI/Blazor/Message.md

```diff
@@ -1,3 +1,101 @@
 # Blazor: UI Message Service
 
-TODO
\ No newline at end of file
+UI message service is used to show nice-looking messages to the user as a blocking dialog.
+
+## Quick Example
+
+Simply inject `IUiMessageService` to your page or component and call the `Success` method to show a success message.
+
+```csharp
+namespace MyProject.Blazor.Pages
+{
+    public partial class Index
+    {
+        public IUiMessageService UiMessageService { get; set; }
```

Comment:
I suppose you forget to [Inject] this. You can use constructor injection since it is supported since the version 4.0

Reply:
In addition, if would be good to mention about the `AbpComponentBase` class that already has a `Message` property (of type `IUiMessageService`) that is pre-injected. So, if you derived from the `AbpComponentBase` class, you can directly use `await Message.Success(...);`.

Reply:
Actually, It works without the [Inject] attribute but I will change the code and use the constructor injection.

Reply:
Oh, it works because ABP (with AutoFac) enables property injection :)